camp/apps/monitors/purpleair/tasks.py

```diff
@@ -16,12 +17,13 @@
 from camp.apps.monitors.purpleair.models import PurpleAir
 
 
-@db_periodic_task(crontab(minute='*/2'), priority=50)
+@db_periodic_task(crontab(minute='*/1'), priority=50)
 def import_recent_data():
     print('[import_recent_data]')
-    for monitor in PurpleAir.objects.all():
-        print('\n' * 10, '[import_recent_data]', monitor.name, '\n' * 10)
-        import_monitor_data.schedule([monitor.pk], delay=1, priority=30)
+    if HUEY.pending_count() < settings.MAX_QUEUE_SIZE:
```

Comment:
I'm generally a fan of early returns, as it makes it explicit that there's nothing left to do in the function. What do you think of changing this to...

```
if HUEY.pending_count() > settings.MAX_QUEUE_SIZE:
    return

for monitor in PurpleAir.objects.all():
    ...
```

?

Reply:
Done.